modules/network-observability-multitenancy.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-multi-tenancy{context}"]
+= Enable multi-tenancy in Network Observability
+Multi-tenancy in the Network Observability Operator allows and restricts individual user access, or group access, to the flows stored in Loki. Access is enabled for project admins. Project admins who have limited access to some namespaces can access flows for only those namespaces.
+
+.Prerequisite
+* You have installed link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.7] 
+* The `FlowCollector` `spec.loki.authToken` configuration must be set to `FORWARD`.
+
+.Procedure
+. Authorize reading permission to `user1` by running the following command: 
++
+[source, terminal]
+----
+$ oc adm policy add-cluster-role-to-user netobserv-reader user1
+----
++
+Now, the data is restricted to only allowed user namespaces. For example, a user that has access to a single namespace can see all the flows internal to this namespace, as well as flows going from and to this namespace.
+The users must be project admins, which means they must have access to the Administrator perspective in the {product-title} console to access the Network Flows Traffic page.
```

Comment:
This might work better as a prereq, like "you must be logged in as an admin" or something along those lines.